consumers/google.go

```diff
@@ -65,10 +56,21 @@ func NewGoogleDNS() (Consumer, error) {
 		return nil, fmt.Errorf("Error creating DNS service: %v", err)
 	}
 
+	resp, err := client.ManagedZones.List(params.project).Do()
```

Comment:
I'll address this in a separate PR. For now I'm fine with this being a limitation.